tensilelite/Makefile

```diff
@@ -0,0 +1,66 @@
+# ########################################################################
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# ########################################################################
+
+
+# ------------------------------------------------------------------------
+#
+# Usage:
+# 
+# Assuming one runs the following Tensile command:
+#
+#   Tensile/bin/Tensile <yaml> tensile-out
+#
+# where tensile-out has the following directory structure:
+#
+# tensile-out
+# ├── 0_Build
+# ├── 1_BenchmarkProblems
+# │   ├── Cijk_Ailk_Bjlk_1
+# |   ├── Cijk_Ailk_Bjlk_2
+# |   ...
+# |   ├── Cijk_Ailk_Bjlk_N
+#
+# One can invoke make with this Makefile as follows:
+#
+# make co PROBLEM_DIR=bar-out/1_BenchmarkProblems/Cijk_Ailk_Bjlk_1
```

Comment:
```suggestion
# make co PROBLEM_DIR=tensile-out/1_BenchmarkProblems/Cijk_Ailk_Bjlk_1
```
I think this should be "tensile-out" to match the example

Reply:
cp paste...